manager.mk

```diff
@@ -28,7 +28,7 @@ BIN           := manager
 # Docker image tag
 IMGTAGS  ?= $(addsuffix /kube-trigger-$(BIN):$(IMG_VERSION),$(REGISTRY))
 
-include makefiles/common-targets.mk
```

Comment:
Can we include the manager and trigger makefiles in the main makefile so that we can directly call the command like `make docker-image-manager`?

Reply:
Sure, made possible.

https://github.com/kubevela/kube-trigger/blob/3817567c93f761023992c2aaf16a3945a64967f4/Makefile#L103-L104

- `make docker-build`: build both manager and trigger containers
- `make docker-build-trigger`: build trigger container
- `make docker-build-manager`: build manager container

Other make targets can also be used in this way.